gitpwnd/bootstrap.py.template

```diff
@@ -177,9 +177,17 @@ def find_root_of_git_repo(path):
         else:
             cur_dir = os.path.join(cur_dir, "..")
 
-# TODO: Iterate through all dirs in base_dir and find all the git repos
+def is_git_directory(path):
+    exit_code = subprocess.call(["git", "status"], cwd=path, stdout=open(os.devnull, 'w'), stderr=subprocess.STDOUT)
```

Comment:
Nice. If we're assuming that `path` is the root of the git directory, then a simpler way may be to just check for the presence of a `path/.git`, which doesn't require subprocess.

The way you're doing it is nice in that if we're in any subdirectory of a git repo it'll still work. Thoughts?